### ISSUE TITLE
Using Jolt as default physics cause error and engine build fails

### ISSUE THREAD

**Issue by CodeLikeCXK** [GaijinEntertainment/DagorEngine#61]
![Dabuild_phys_joltasdefault](https://github.com/GaijinEntertainment/DagorEngine/assets/44729188/2e7e9b70-696c-4fa3-90dd-dea846d40e8b)

Seems some editor tools relies on bullet?

**Comment by NicSavichev** [comment]
Yes, tools cannot use Jolt so if you change the default setting you have to explicitly change PhysName for daEditorX to Bullet as well

**Comment by CodeLikeCXK** [comment]
> Yes, tools cannot use Jolt so if you change the default setting you have to explicitly change PhysName for daEditorX to Bullet as well

I see...so how to change physname for tools excplicitly?

**Comment by CodeLikeCXK** [comment]
> Yes, tools cannot use Jolt so if you change the default setting you have to explicitly change PhysName for daEditorX to Bullet as well

Sorry. Seems physics engine for editor (PhysName = Bullet) should be set in tools_setup.jam file. Thanks!

**Comment by NicSavichev** [comment]
fixed in latest https://github.com/GaijinEntertainment/DagorEngine/releases/tag/dagor_2024_12_04
